docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[ml-configuring-alerts]]
+= Configuring {anomaly-detect} alerts
+
+{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
+of jobs to detect anomalies with certain conditions. If an {anomaly} meets the 
+conditions, the alert triggers the defined action. For example, you can create 
+an alert that checks an {anomaly-job} in every fifteen minutes for critical 
+anomalies and notify you in an email. This page helps you to configure an 
+{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
+
+
+[[creating-anomaly-alerts]]
+== Creating an alert
+
+{anomaly-detect-cap} alerts can be created in the {anomaly-job} wizard, from the 
+job list or under **Stack management > Alerts and Actions**. On the *Create 
+alert* template, select *{anomaly-detect-cap} alert* under the {ml-cap} section, 
+then give a name to the alert and optionally provide tags.
+
+[role="screenshot"]
+image::images/ml-anomaly-alert-type.jpg["Creating an anomaly detection alert"]
+
+Specify the time interval for the alert to check detected anomalies. It is 
+recommended to select an interval that is similar to the bucket span of the 
+associated job. You can also select a notification option by using the _Notify_ 
+selector. There are three notification options:
```

Comment:
> There are three notification options:

This also seems to be standard behaviour for all alerts. Is it worthwhile defining them here?  If we put these definitions in https://www.elastic.co/guide/en/kibana/master/defining-alerts.html#defining-alerts-general-details instead, perhaps we don't need to restate them here?

Reply:
I remove these options from here and add them to the defining alerts page in a separate PR.